data_structures/stacks/evaluate_postfix_notations.py

```diff
@@ -17,32 +17,52 @@ def evaluate_postfix(postfix_notation: list) -> int:
     9
     >>> evaluate_postfix(["4", "13", "5", "/", "+"])
     6
+    >>> evaluate_postfix(["2", "+"])
+    2
+    >>> evaluate_postfix(["5", "-"])
+    -5
     >>> evaluate_postfix([])
     0
     """
     if not postfix_notation:
         return 0
 
     operations = {"+", "-", "*", "/"}
+    unary_operations = {"+"}  # Unary operator(s)
```

Comment:
`unary_operations` is not currently being used.

Reply:
now I made several changes, kindly look into it.